homeassistant/components/maxcube/climate.py

```diff
@@ -144,13 +144,10 @@ def set_hvac_mode(self, hvac_mode: str):
         """Set new target hvac mode."""
         device = self._cubehandle.cube.device_by_rf(self._rf_address)
         temp = device.target_temperature
-        mode = device.mode
```

Comment:
@MartinHjelmare To answer your question about the manual mode. It is set as a default.

`mode` = always set via the else statement. Hence this line originally was useless. Setting a proper default here, made the `elif hvac_mode == HVAC_MODE_HEAT:` unneeded.

Reply:
Oh shit you are right